app/internal/load_jokes.py

```diff
@@ -0,0 +1,38 @@
+import json
+from typing import Dict, List, Optional
+
+from app.database.models import Joke
+
+from sqlalchemy.orm import Session
+
+
+def get_jokes_from_json() -> List[Dict[str, Optional[str]]]:
+    """Reading all jokes from the a JSON file that have been """\
+        """copied from 'http://api.icndb.com/jokes'"""
+    try:
+        with open('app/resources/jokes.json', 'r') as f:
+            jokes_list = json.load(f)
+    except (IOError, ValueError):
+        return []
+    return jokes_list
```

Comment:
isnt that another pr?

Reply:
Yes. 
I hope that I succeed to separate them again